app/views/local_transaction/devolved_administration_service.html.erb

```diff
@@ -0,0 +1,24 @@
+<%= render layout: 'shared/base_page', locals: {
+  title: @publication.title,
+  publication: @publication,
+  edition: @edition,
+} do %>
+  <div class="interaction">
+    <p class="govuk-body">
+      <%= "Get information on this service, or a similar service, on the #{@country_name} Government website."%>
+    </p>
+    <div class="local-authority-result"
+         data-module="auto-track-event"
+         data-track-category="userAlerts:local_transaction"
+         data-track-action="ResultsShownAvailableDa">
```

Comment:
I changed the value to start with a lower case character. I did suggest some values to Harriet but she chose the ones I added so I think we should keep them? Otherwise I can try asking her what she thinks about those options?

Reply:
Yeah I think we should check these suggestions first before or at least give Harriet an opportunity to refine a bit. I'll drop a quick email on the thread to see how these go.